pangeo-notebook/packages.txt

```diff
@@ -122,6 +122,7 @@ docopt==0.6.2
 docrep==0.3.2
 docutils==0.15.2
 donfig==0.7.0
+earthaccess==0.5.2
 earthdata==0.2.2
```

Comment:
thanks for catching this @weiji14, to *remove* packages unfortunately you have to also wipe the existing conda-lock.yml (https://github.com/conda/conda-lock/issues/196)

Reply:
I see, maybe it would be helpful to add an `rm conda-lock.yml` command after `cd ${{ matrix.IMAGE }}` here https://github.com/pangeo-data/pangeo-docker-images/blob/9375b20d1f4c893df43d4703c35029db238e4ee5/.github/workflows/CondaLock.yml#L40-L47?

Reply:
i suppose so. although, i'm not sure what the recommended best practices are here... for example, does the solver give a different result or is it faster if updating an existing conda-lock.yml? the only other thing i can think of is the github diff the entire long file versus line changes.

Reply:
Not sure if the solver is faster. Maybe we should ask on https://github.com/conda/conda-lock/issues/196 to check if the issue has been solved with conda-lock>=2?